Closes #17

machado-andre opened the pull request:
Review Thoroughly if you can. Much appreciated

machado-andre committed:
Breeds Pagination

Breeds API Call for each page has page buttons associated.
Code is essentially, WHACK. but for now it works.

#TODO
Have Method callings available to receive params.

machado-andre committed:
Fix #17

Hardcode = ew
DYNAMIC!

eusouorui reviewed (changes requested):
On hover buttons get out of page
![image](https://github.com/eusouorui/CatFact/assets/102189045/559e8b29-59bb-464a-98f6-3070893bef95)

eusouorui commented:
Hover problem still occurs.
Try forcing the same css properties on hover.

machado-andre committed:
Fix #17

Removed i and index confusion.
Insanity of a ternary if <3

machado-andre committed:
Closes #17

Changed Table HTML tag to ASPNetCore QuickGrid Component to list all breeds making the list queriable.
Improved Pagination UI +-
Should be able to close this Branch Issue.

eusouorui commented:
@machado-andre Code does not compile due to errors:
<img width="1285" alt="image" src="https://github.com/eusouorui/CatFact/assets/102189045/45782fcd-fc7d-40d1-8301-0ae5782a1f47">

Please fix
